doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,101 @@
+.. _ug_bt_qualification:
+
+Bluetooth qualification
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Every product using the Bluetooth® LE technology must be certified (qualified) by Bluetooth Special Interest Group (SIG).
+Only when your product is qualified, you will be authorized to do the following:
+
+* Prove that your products satisfy the requirements of the Bluetooth license agreements and specifications.
+* Use Bluetooth trademark and logos.
+* Sell products that use Bluetooth LE technology.
+
+As a result, your products will be listed and can be promoted on the official Bluetooth SIG page.
+
+Pre-qualified design
+********************
+
+.. note::
+   A *design* is a specific configuration of hardware or software implementation of adopted Bluetooth specifications.
+
+By using pre-qualified designs of products, you can omit the testing phase during your end product's qualification.
+
+Nordic Semiconductor uses all available `Bluetooth product types`_. Some of them can be qualified by yourself.
+All versions of End products, Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems, and Components are created and qualified by Nordic Semiconductor by default.
+Profile Subsystems (for example, :ref:`ug_bt_mesh` versions) can be qualified by any party.
+If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_ yet, run tests through the `Profile Tuning Suite (PTS)`_ tool and qualify the product.
+
+Qualification process
+*********************
+
+The product qualification cannot be inherited, hence, you must complete this process by yourself.
+You can either perform a full qualification process for a new product or edit an existing entry for a product that has already been qualified.
+
+Creating new entry
+==================
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following processes depending on the state of your product:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
+* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.
+
+Updating existing entry
+=======================
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You significantly changed an existing product's design so that the testing is required.
```

Comment:
Why do you say that? There are two paths in the qualification process. 1. Requires testing 2. Does not require testing. Both of these paths can be used to create a "new entry" or "update existing entry".

Can you make this clear by adding the necessary bullets?

Reply:
Ok